llvm/cmake/config-ix.cmake

```diff
@@ -117,7 +117,11 @@ endif()
 
 if(LLVM_ENABLE_ZLIB)
   if(LLVM_ENABLE_ZLIB STREQUAL FORCE_ON)
-    find_package(ZLIB REQUIRED)
+    if("${ZLIB_ROOT}" STREQUAL "")
+      find_package(ZLIB REQUIRED)
+    else()
+      include(FindZLIB)
+    endif()
```

Comment:
So in https://reviews.llvm.org/D70519 there was an attempt to use FindZLIB and it was noted it changes the value in `llvm-config --system-libs`, is that still the case?

Reply:
It will select `$ZLIB_ROOT/libz.so` which is enough for our purposes. As long as RPATH is correct (which I haven't checked as full cross-build was taking ages on my machine), it will load on any system with libz. On NativeAOT, libz is a system prerequisite on build machine, so ilc will be ok.